gammapy/makers/map.py

```diff
@@ -170,10 +171,22 @@ def make_exposure(geom, observation, use_region_center=True):
             return observation.exposure.interp_to_geom(
                 geom=geom,
             )
-        if isinstance(observation.aeff, Map):
-            return observation.aeff.interp_to_geom(
-                geom=geom,
-            )
+        with u.add_enabled_units([u.def_unit("transit", u.sday)]):
+            if isinstance(observation.aeff, Map):
+                aeff = observation.aeff
+                if aeff.unit.is_equivalent(u.Unit("m2 s")):
+                    factor = 1.0
+                elif aeff.unit.is_equivalent(u.Unit("m2 s transit-1")):
+                    if u.Unit("transit") in aeff.unit.bases:
+                        factor = observation.gti.time_sum.to("transit")
```

Comment:
maybe put the TODO here too so that it doesn't slip past?

Reply:
added TODO